lib/smart_proxy_ansible/reader_helper.rb

```diff
@@ -32,7 +32,7 @@ def path_from_config(config_key)
 
         def playbook_or_role_full_name(path)
           parts = path.split('/')
-          playbook = parts.pop
+          playbook = parts.pop.split('.yml').first
```

Comment:
Cannonical extensions for yaml files are either `yml` or `yaml`. Would be nice if we could cover both here